It seems to be running now, stay tuned... 

However, when I first tried to run it I was missing an fgmax input file in this directory and so it threw a fortran error, and several Python errors in the process of doing so. Maybe this is the expected behavior in this case, but it may make it harder for the user to figure out what the real error was.  Is it possible to print out a more useful message saying the fortran code died rather than looking like a Python error in this case?

(Note to self: change `old` to `unknown` in opening the fgmax file from `fgmax_read.f90` so this prints the message it is supposed to rather than throwing a fortran error.)

```
Reading data file: fgmax.data
         first 5 lines are comments and will be skipped
At line 47 of file /Users/rjl/clawpack_src/clawpack_master/geoclaw/src/2d/shallow/fgmax_read.f90 (unit = 45)
Fortran runtime error: Cannot open file '/Users/rjl/git/WA_EMD_2018/RANDY/bainbridge/eagle_harbor_test_faster_filpatch/fgmax_eagle_harbor.txt': 
  No such file or directory

Error termination. Backtrace:
#0  0x1086c46fd
#1  0x1086c5395
#2  0x1086c5b29
#3  0x108889968
#4  0x108889c5c
#5  0x108680b8f
#6  0x10863c79c
#7  0x10867fbcf
#8  0x10868f21e
Traceback (most recent call last):
  File "/Users/rjl/clawpack_src/clawpack_master/clawutil/src/python/clawutil/runclaw.py", line 227, in runclaw
    stderr=xclawerr)
  File "/Users/rjl/miniconda/envs/geo/lib/python3.6/subprocess.py", line 291, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['/Users/rjl/git/WA_EMD_2018/RANDY/bainbridge/eagle_harbor_test_faster_filpatch/xgeoclaw']' returned non-zero exit status 2.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/rjl/clawpack_src/clawpack_master/clawutil/src/python/clawutil/runclaw.py", line 322, in <module>
    runclaw(*args)
  File "/Users/rjl/clawpack_src/clawpack_master/clawutil/src/python/clawutil/runclaw.py", line 232, in runclaw
    stderr=cpe.stderr)
__main__.ClawExeError: error
make: *** [output] Error 1

```

--- a/src/python/clawutil/runclaw.py
+++ b/src/python/clawutil/runclaw.py
@@ -227,6 +227,9 @@ def runclaw(xclawcmd=None, outdir=None, overwrite=True, restart=None,
                                      stderr=xclawerr)
 
     except subprocess.CalledProcessError as cpe:
+        print("")
+        print("*** FORTRAN EXE FAILED ***")
+        print("")
         raise ClawExeError('error', cpe.returncode, cpe.cmd,
                            output=cpe.output, 
                            stderr=cpe.stderr)

--- a/src/python/clawutil/runclaw.py
+++ b/src/python/clawutil/runclaw.py
@@ -227,10 +227,8 @@ def runclaw(xclawcmd=None, outdir=None, overwrite=True, restart=None,
                                      stderr=xclawerr)
 
     except subprocess.CalledProcessError as cpe:
-        print("")
-        print("*** FORTRAN EXE FAILED ***")
-        print("")
-        raise ClawExeError('error', cpe.returncode, cpe.cmd,
+        exe_error_str = "\n\n*** FORTRAN EXE FAILED ***\n"
+        raise ClawExeError(exe_error_str, cpe.returncode, cpe.cmd,
                            output=cpe.output, 
                            stderr=cpe.stderr)
     

